### PR TITLE
Create valid XML for mrefd log in DHT branch

### PR DESCRIPTION
**Pull request by kc1awv**
The current logging scheme for mrefd does not produce valid XML since currently there are spaces in the tags.

I have reformatted the tags to be XML compliant.

This may be a breaking change for existing mrefd dashboards, so let's keep these changes in the DHT branch for now, while a new dashboard system is developed.

### CODE REVIEW

File: reflector.cpp [hunk 1]
```diff
@@ -443,5 +446,8 @@ void CReflector::WriteXmlFile(std::ofstream &xmlFile)
 	}
 	// unlock
 	ReleaseUsers();
-	xmlFile << "</" << m_Callsign << " heard users>" << std::endl;
+	xmlFile << "</STATIONS>" << std::endl;
+	
+	// reflector end
+	xmlFile << "</REFLECTOR>" << std:endl;
```
Comment (dbehnke) [inline reflector.cpp:452]: you forgot a colon (should be double) .. std::endl;

File: reflector.cpp [hunk 2]
```diff
@@ -398,13 +398,16 @@ void CReflector::WriteXmlFile(std::ofstream &xmlFile)
 	// write header
 	xmlFile << "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" << std::endl;
 
+	// reflector start
+	xmlFile << "<REFLECTOR CALLSIGN=\">" << m_Callsign << "\">" << std:endl;
```
Comment (dbehnke) [inline reflector.cpp:402]: you need double :: here.. std::endl;

**Review by kc1awv — COMMENTED** [review]
Typo fixes, thanks Dave!

File: reflector.cpp (hunk 1 above)
Comment (kc1awv) [reply reflector.cpp]: Patched, thanks Dave!

File: reflector.cpp (hunk 2 above)
Comment (kc1awv) [reply reflector.cpp]: Patched, thanks Dave!

**Pull request merged**